app/controllers/administrative-units/administrative-unit/sites/new.js

```diff
@@ -20,6 +21,14 @@ export default class AdministrativeUnitsAdministrativeUnitSitesNewController ext
     );
   }
 
+  get isAndereVestigen() {
```

Comment:
This getter should be placed in the `site` model such that it does not need to be duplicated here and in the edit route.  (This is a change coming with new form validation framework.)
Concerning the function name, I am inclined to use English names to avoid (imo) unnecessary hurdles for non Dutch speakers to read/understand the code, so in this case something like `isOtherSite` seems appropriate. If you do want to keep the Dutch name it should be `isAndereVestiging` as "Vestig**en**" is not correct in this context.

Reply:
Done :)